Insight Docs Tweaks (#4263)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/insights/insights-view/index.md

techPreview: true
---

> <p class="message--note"><strong>NOTE: </strong>Insights are refreshed on a 72-hour basis. Older insights expire, however, if an insight persists, it reappears in the table.</p>

Reviewer: ```suggestion
<p class="message--note"><strong>NOTE: </strong>Insights are refreshed on a 72-hour basis. Older insights expire. However, if an insight persists, it reappears in the table.</p>
```

> <p class="message--note"><strong>NOTE: </strong>Insights are refreshed on a 72-hour basis. Older insights expire, however, if an insight persists, it reappears in the table.</p>

Reviewer 2: ```suggestion
<p class="message--note"><strong>NOTE: </strong>Insights are refreshed every 72-hours. Older insights expire, but if an insight persists, it reappears in the table.</p>
```

> After the [Insights Engine is enabled](../insights-setup/), select a workspaces to view your insight items. On the Workspace Dashboard below the workspace summary cards, the Insights summary card displays the most recent insight items. The Insights summary card displays the amount of insights within each severity level of Critical, Warning, and Notices. Select the View All link to see a summary of all insight items on the Insights Dashboard.

Reviewer: ```suggestion
After the [Insights Engine is enabled](../insights-setup/), select a workspace to view your insight items. On the Workspace Dashboard below the workspace summary cards, the Insights summary card displays the most recent insight items. The Insights summary card displays the amount of insights within each severity level of Critical, Warning and Notices. Select the View All link to see a summary of all insight items on the Insights Dashboard.
```

> Select **Insights** from the sidebar menu for the DKP Insights Dashboard, where you can filter the insight items several different ways:

Reviewer 3: I think this is the standard that we call it?

```suggestion
Select **Insights** from the side nav menu for the DKP Insights Dashboard, where you can filter the insight items several different ways:
```
